packages/scripts/README.md

```diff
@@ -0,0 +1,125 @@
+
+```{note}
+It's quite similar when you develop a Plone backend add-on in the Python side, and embed a ready to use Plone build (using buildout or pip) in order to develop and test the package.
+```
+
+Unfortunatelly, the NodeJS tooling does not work well with symlinks or outside the root of the project.
+This script proceeds in the following way:
+
+1. Generates a vanilla project using the official Volto Yo Generator (@plone/generator-volto)
+2. Configures is to use the add-on with the name stated in the `package.json`
+3. Copies over the root of the add-on (`src` and essential files) inside the created project
+
+After this, you can change directory to the created project, by default `addon-testing-project`.
+The name of the created project is parameterizable.
+
+After that you can use the full fledged project, and run any standard Volto command for linting, acceptance test or unit tests.
+
+### clone (git)
+
+Given the add-on remote git repository it pulls and configures it into the vanilla project generated by the script.
+
+`npx -p @plone/scripts addon clone [options] <source> [destination]`
+
+    Usage: addon clone [options] <source> [destination]
+
+    clone a repository into a newly created directory
+
+    Options:
+      -p, --private          set if the repo is private, then GITHUB_TOKEN is used
+      -b, --branch <branch>  set the repo branch, defaults to main
+      -c, --canary           downloads latest Volto canary (alpha) version
+      -h, --help             display help for command
+
+This next command, would download the `volto-blocks-grid` add-on from its git repo, branch `main` and will generate a project with the latest Volto canary (alpha) version.
+
+```shell
+npx -p @plone/scripts addon clone https://github.com/kitconcept/volto-blocks-grid.git --branch main --canary
+```
+
+This will create a directory named `addon-testing-project` and will bootstrap a new project using Volto's standard project generator.
+It will adjust the configuration of this project to setup the add-on in the project using `mrs-developer` and the git URL given to fetch the add-on contents.
+You can specify the branch to be used, if the project should use the latest alpha available.
+There is an option for private repos as well, in that case, it will use the `GITHUB_TOKEN` present in your environment variables to fetch it.
+
+After this, as a developer you can use the usual project commands to run tests (unit, linting, acceptance) inside the generated `addon-testing-project`.
+You can configure the CI of your choice for automated testing, you can take a look at how it's done in: https://github.com/kitconcept/volto-blocks-grid/tree/main/.github/workflows
+
+The idea is to issue commands inside the generated `addon-testing-project` project and do your checks.
+Take special care on how to pass down to the `npx` command the current PR branch depending on your CI system this might be different.
```

Comment:
```suggestion
Take special care on how to pass down to the `npx` command the current pull request branch.
Depending on your CI system, this might be different.
```